src/components/Branding/styled-components.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import { color } from '../common-styled-components';
+
+export const BrandingContainer = styled.div`
+  position: fixed;
+  top: 0px;
+  width: inherit;
+  overflow-wrap: break-word;
+  text-align: center;
+  font-size: 28px;
+  letter-spacing: -2px;
+  margin-top: 8px;
+  margin-left: -2px;
+  span:nth-child(1) {
+    color: ${color.blue};
+    font-weight: bold;
+  }
+  span:nth-child(2) {
+    color: ${color.orange};
+    font-weight: bold;
```

Comment:
I feel like if you're gonna repeat "font-weight" on 2 lines, you should attempt it on a parent div to see if it applies to both. Keep code DRY

Reply:
Moved font-weight to parent